### DISCUSSION TITLE
when to support leidenalg?

### DISCUSSION THREAD

**Discussion by hugofgh** [networkx/networkx#5385] [Algorithms]
when to support leidenalg?

**Comment by MridulS** [comment]
We currently don't have leiden algorithm implemented inside networkx and there are no specific plans to implement it right now.

From now people can convert their networkx graph objects to igraph graph and use https://github.com/vtraag/leidenalg to find communities using the leiden algorithm.

**Comment by dschult** [comment]
Also, would you like to work on creating  python version of leidenalg for inclusion in NetworkX?
We would be open to such a submission.
